proxy/src/main/java/net/md_5/bungee/command/ConsoleCommandCompleter.java

```diff
@@ -24,7 +24,7 @@ public int complete(String buffer, int cursor, List<CharSequence> candidates)
             String lowerCase = buffer.toLowerCase( Locale.ROOT );
             candidates.addAll( proxy.getPluginManager().getCommands().stream()
                     .map( Map.Entry::getKey )
-                    .filter( (name) -> name.toLowerCase( Locale.ROOT ).startsWith( lowerCase ) )
+                    .filter( name -> name.toLowerCase( Locale.ROOT ).startsWith( lowerCase ) )
```

Comment:
Why? Code style prefers paren

Reply:
I don't relly know why i changed it i think it looks prittier but i will change it back